client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,183 @@
+
+export default function ManageBookings(): JSX.Element {
+  const classes = useStyles();
+  const [selectedDate, setSelectedDate] = useState<Date | null>(new Date());
+  const [requestDates, setRequestDates] = useState<(string | null)[]>();
+  const [nextBookingRequest, setNextBookingRequest] = useState<BookingRequest>();
+  const [pastBookingRequests, setPastBookingRequests] = useState<BookingRequest[]>();
+  const [currentBookingRequests, setCurrentBookingRequests] = useState<BookingRequest[]>();
+  const [nextBookingDisplay, setNextBookingDisplay] = useState<JSX.Element>();
+  const [currentBookingsDisplay, setCurrentBookingsDisplay] = useState<JSX.Element>();
+  const [pastBookingsDisplay, setPastBookingsDisplay] = useState<JSX.Element>();
+
+  const updateBookingDisplay = (bookingRequestName: any, time: string) => {
+    const methods: any = {
+      next: setNextBookingDisplay,
+      current: setCurrentBookingsDisplay,
+      past: setPastBookingsDisplay,
+    };
+    // if bookingRequestName is array, map the booking requests (current and past)
+    if (bookingRequestName && bookingRequestName.length >= 1) {
+      methods[time](
+        <>
+          {bookingRequestName.map((booking: BookingRequest) => {
+            const profile = booking.user_id.profile;
+            return (
+              <Grid key={booking._id}>
+                <Booking
+                  outlined
+                  accepted={!booking.accepted && !booking.declined ? undefined : booking.accepted}
+                  name={`${profile && profile.firstName} ${profile && profile.lastName}`}
+                  avatar={profile && profile.profileImg}
+                  date={booking.start_date}
+                />
+              </Grid>
+            );
+          })}
+        </>,
+      );
+      // if bookingRequestName is not an array, display the single booking request (next booking request)
+    } else if (bookingRequestName) {
+      const profile = bookingRequestName.user_id.profile;
+      methods[time](
+        <>
+          <Grid key={bookingRequestName._id}>
+            <Booking
+              outlined
+              accepted={
+                !bookingRequestName.accepted && !bookingRequestName.declined ? undefined : bookingRequestName.accepted
+              }
+              name={`${profile && profile.firstName} ${profile && profile.lastName}`}
+              avatar={profile && profile.profileImg}
+              date={bookingRequestName.start_date}
+            />
+          </Grid>
+        </>,
+      );
+    }
+  };
+
+  const getRequests = async () => {
+    const data = await getBookingRequests();
+    const currentRequestsList: BookingRequest[] = [];
+    const pastRequestsList: BookingRequest[] = [];
+    const requestDatesList: (string | null)[] = [];
+    const now = new Date().valueOf();
+    if (data.requests) {
+      data.requests.map((request) => {
+        let requestDate = request.start_date.split('T')[0];

```

Comment:
Same here, could this be changed to be a bit better? Also, please use `camelCase` and not `snake_case`

Reply:
The snake_case came from a different PR -- I just didn't want to refactor the entire controller that I didn't make. I can change it if necessary though.

That split indeed wasn't necessary there. Removed.